src/scenarios/init.ps1

```diff
@@ -1,24 +1,45 @@
 Param(
-    $Channel = 'master' # default is master
+    [string] $InstallDotnetFromChannel,
+    [string] $DotnetDirectory
 )
 
+# Add scripts and current directory to PYTHONPATH
 $scripts = Join-Path $PSScriptRoot '..\..\scripts' -Resolve
 $env:PYTHONPATH="$scripts;$PSScriptRoot"
-$dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
-$dotnetDirectory = Join-Path $PSScriptRoot '..\..\tools\dotnet\x64'
 
-Write-Host "Downloading dotnet from channel $Channel"
-python $dotnetScript install --channels $Channel -v
+If (($InstallDotnetFromChannel -ne "") -and ($DotnetDirectory -eq "")){
+    # Download dotnet from the specified channel
+    
+    $installDirectory= Join-Path $PSScriptRoot 'dotnet'
+    # Remove existing dotnet directory to make sure we only have one version of dotnet
+    If (Test-Path $installDirectory){
+        Write-Host "Removing $installDirectory ..."
+        Remove-Item  $installDirectory -Recurse
+    }
 
-if (!$?){
-    Write-Host "Dotnet installation failed."
-    Exit 1
+    Write-Host "Downloading dotnet from channel $InstallDotnetFromChannel"
+    $dotnetScript= Join-Path "$scripts" 'dotnet.py' -Resolve
+    python $dotnetScript install --channels $InstallDotnetFromChannel --install-dir $installDirectory
+
+    If (!$?){
+        Write-Host "Dotnet installation failed."
+        Exit 1
+    }
+}
+
+If ($DotnetDirectory -ne ""){
```

Comment:
You could simplify this by setting `$DotnetDirectory` based on what the user chose and then just add that to the path always.